common/src/services/state.service.ts

```diff
@@ -2283,7 +2277,7 @@ export class StateService<TAccount extends Account = Account>
     return {
       storageLocation: StorageLocation.Disk,
       htmlStorageLocation: HtmlStorageLocation.Session,
-      userId: await this.getActiveUserIdFromStorage(),
+      userId: this.state.activeUserId ?? (await this.getActiveUserIdFromStorage()),
```

Comment:
This should cut down on the unnecessary calls to get the userId from storage every time a setting is loaded. Nice!